### DISCUSSION TITLE
Documentation link

### DISCUSSION THREAD

**Discussion by strobeflash** [GothenburgBitFactory/taskwarrior#2786] [General]
While reading through the docs, I noticed that the link to tasksh at the bottom of the page points to a news article from 2016. (https://taskwarrior.org/news/news.20160905.2.html)

Maybe a link to the actual docs page would be more helpful, would have been for me anyway.

https://taskwarrior.org/docs/best-practices.html#review

I couldn't find where the documentation lies though, how do I create a pull request for this?

**Comment by djmitche** [comment]
The docs are at https://github.com/GothenburgBitFactory/tw.org but are currently in a state of flux as they are transformed from raw HTML into a markdown-based format.  There are a number of issues similar to the one you've identified that will be much easier to solve once that's complete.

**Comment by strobeflash** [comment]
Thanks, created a pull request for this.
https://github.com/GothenburgBitFactory/tw.org/pull/388
